.github/workflows/superlinter.yml

```diff
@@ -16,9 +21,9 @@ jobs:
           fetch-depth: 0
 
       - name: Lint Code Base
-        uses: docker://github/super-linter:slim-v4
+        uses: github/super-linter@v4
         env:
-          DEFAULT_BRANCH: main
+          DEFAULT_BRANCH: master
```

Comment:
there is not master branch in repo.

Reply:
DEFAULT_BRANCH | master | The name of the repository default branch.
-- | -- | --